lib/src/screens/session/widgets/issuance_permission.dart

```diff
@@ -44,16 +44,13 @@ class IssuancePermission extends StatelessWidget {
       padding: EdgeInsets.symmetric(horizontal: theme.defaultSpacing),
       children: [
         Padding(
-          padding: EdgeInsets.symmetric(vertical: theme.smallSpacing),
-          child: Container(
-            color: theme.surfaceSecondary,
-            padding: EdgeInsets.all(theme.defaultSpacing),
-            child: TranslatedText(
-              'issuance.description',
-              style: theme.textTheme.caption,
-            ),
-          ),
-        ),
+            padding: EdgeInsets.symmetric(vertical: theme.smallSpacing),
+            child: IrmaQuote(
+              quote: FlutterI18n.translate(
+                context,
+                'issuance.description',
+              ),
+            )),
```

Comment:
Nit pick: a comma and a newline between the two parentheses might be nice to fix the indents.